tests/test_app/library/books/receivers.py

```diff
@@ -0,0 +1,12 @@
+from django.contrib.auth.models import User
+from django.db.models.signals import pre_save
+from django.dispatch import receiver
+
+
+@receiver(pre_save, sender=User)
```

Comment:
Shouldn't it be in a `post_save` signal instead?

Reply:
nope, this is setting the password in on the instance before it is saved, using post_save would cause a double save